tests/distributions/test_multivariate.py

```diff
@@ -330,6 +330,41 @@ def test_mvnormal_init_fail(self):
             with pytest.raises(ValueError):
                 x = pm.MvNormal("x", mu=np.zeros(3), cov=np.eye(3), tau=np.eye(3), size=3)
 
+    @pytest.mark.parametrize("batch_mu", (False, True))
+    @pytest.mark.parametrize("batch_cov", (False, True))
+    @pytest.mark.parametrize("cov_form", ("cov", "tau", "chol_lower", "chol_upper"))
```

Comment:
@fonnesbeck now testing `chol` parametrizations as well. That was an oversight on my end :)